kano_updater/apt_wrapper.py

```diff
@@ -355,8 +355,8 @@ def fix_broken(self, progress):
         if self._cache._depcache.broken_count:
             try:
                 self._cache._depcache.fix_broken()
-            except SystemError as err:
-                logger.error(err)
+            except SystemError as e:
+                logger.error('Error attempting to fix broken pkgs', exception=e)
```

Comment:
I've seen us doing it like this at several places: `logger.error('ouch: {}'.format(e))`, are both methods valid?

Reply:
👍